distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -10,15 +10,15 @@ To exit this mode, click the `?` again or press `escape`.
 
 === Accessing ${catalog-ui}
 
-The default location for the ${catalog-ui} is ${secure_url}/search/catalog
+The default location for ${catalog-ui} is ${secure_url}/search/catalog
 
 .Catalog UI Guest Users
```

Comment:
it's the caption for the `NOTE` block. In the final HTML, it renders like this: 

![screen shot 2017-05-04 at 8 16 24 am](https://cloud.githubusercontent.com/assets/10619999/25710772/1ede81d4-30a2-11e7-92ec-a204c1a97a23.png)
It's more _subtle_ in the pdf, but at least it's italicized: 

![screen shot 2017-05-04 at 8 18 34 am](https://cloud.githubusercontent.com/assets/10619999/25710854/5ca352a6-30a2-11e7-9a14-8df8d9286e23.png)